install.sh

```diff
@@ -106,7 +106,7 @@ echo "Sign into the Steam account that owns FF7 if prompted."
 nohup steam steam://rungameid/39140 &> /dev/null &
 echo "Waiting for Steam..."
 while ! pgrep "FF7_Launcher" > /dev/null; do sleep 1; done
-pkill -9 "FF7_Launcher"
+killall -9 "FF7_Launcher"
```

Comment:
This didn't work on my machine unless I added `.exe` to the search string

Reply:
> This didn't work on my machine unless I added `.exe` to the search string

Oops, you are right, I took your word at face value last time without testing.

Reply:
Not sure what you mean by this, I mentioned needing the extension in my first reply.

Reply:
I can't read it seem :)